lms/envs/common.py

```diff
@@ -61,6 +61,7 @@
 }
 
 LMS_ROOT_URL = "http://localhost:8000"
+ENTERPRISE_API_URL = LMS_ROOT_URL + '/enterprise/api/v1/'
```

Comment:
I _think_ that we are supposed to avoid putting actual values in common.py and instead specify in the pertinent environment modules or even lms.env.json through the DevOps provisioning pipeline.